bridge/client/app/_views/ktb-settings-view/ktb-integration-view/ktb-modify-uniform-subscription/ktb-modify-uniform-subscription.component.spec.ts

```diff
@@ -320,6 +328,66 @@ describe('KtbModifyUniformSubscriptionComponent', () => {
     });
   });
 
+  it('should not show a notification when the component is initialized', () => {
+    // given
+    fixture.detectChanges();
+
+    // then
+    // Has to be retrieved by document, as it is not created at component level
+    const notifications = document.getElementsByTagName('dt-confirmation-dialog-state');
+    expect(component.unsavedDialogState).toBeNull();
+    expect(notifications.length).toEqual(0);
+  });
```

Comment:
This should be moved to Cypress. Evaluating HTML elements should not be part of the unit tests.